datahub-frontend/app/utils/ControllerUtil.java

```diff
@@ -29,4 +33,26 @@ public static boolean checkErrorCode(Exception e, int statusCode) {
     return (e instanceof RestLiResponseException) && (((RestLiResponseException) e).getStatus() == statusCode);
   }
 
+  /**
+   * Creates a generic jsonNode out of a key and value which is a very common use case
+   * @param key
+   * @param value
+   * @return
+   */
+  @Nonnull
+  public static JsonNode jsonNode(@Nonnull final String key, @Nonnull final Object value) {
```

Comment:
We use pegasus, as does restli, right? Which has json conversion? Why are we building json ourselves where?

Reply:
As discussed offline, I will not address this because this is coming from internal codebase as is. We should do this refactoring in the internal codebase first. Just to unblock things in favor of pushing `datahub-web` code, I will accelerate merging this PR.